odl/discr/grid.py

```diff
@@ -26,7 +26,7 @@
 
 from future import standard_library
 standard_library.install_aliases()
-from builtins import range, super, zip
+from builtins import range, super, str, zip
```

Comment:
[here](https://github.com/odlgroup/odl/blob/issue-246__l2discr/odl/discr/grid.py#L129). It's a safe import and makes the code slightly more predictable.